automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/Modals.scala

```diff
@@ -54,6 +54,10 @@ case class MessageModal(implicit webDriver: WebDriver) extends OKCancelModal {
     testId("message-modal-content").element != null
   }
 
+  def readMessageModalText: String = {
```

Comment:
The implementations of `readMessageModalText` and `getMessageText` are identical. We should keep one and eliminate the other. I would vote to keep `readMessageModalText` (unless one of them is actually not used anywhere in which case that one should just be removed).